server/httpServer.cpp

```diff
@@ -47,12 +51,50 @@ HttpServer::HttpServer(int pPort) {
     CROW_ROUTE(app, "/version").methods("GET"_method)
         ([](const crow::request& req){
             json j;
-            j["status"] = 200;
+            j["status"] = 200;  
             j["version"] = VERSION;
             j["lastCompiled"] = fmt::format("{} {}", __DATE__, __TIME__);
             return to_string(j);
         });
 
+    CROW_ROUTE(app, "/authtest").methods("GET"_method)
+    .CROW_MIDDLEWARES(app, AuthMiddleware)
+        ([](const crow::request& req){
+            json j;
+            j["status"] = 200;  
+            j["version"] = VERSION;
+            j["lastCompiled"] = fmt::format("{} {}", __DATE__, __TIME__);
+            return to_string(j);
+        });
+
+    CROW_ROUTE(app, "/login").methods("POST"_method)
+        ([](const crow::request& req){
+            json j;
+            auto body = crow::json::load(req.body);
+            std::string name = std::string(body["name"]);
+            std::string pw = std::string(body["pw"]);
+            lowercize(name, 1);
+
+            if (!body) {
+                return crow::response(crow::status::BAD_REQUEST);
+            }
+
+            Player *player = nullptr;
+            if (!loadPlayer(name, &player)) {
+                return crow::response(crow::status::NOT_FOUND);
+            }
+
+            if (!player->isPassword(pw)) {
```

Comment:
Just a heads up, I'll fix it - player is never cleaned up so it'd be a memory leak.

`cmdFinger` would be a good command as an example -- if the player's online it uses that, if they're not it loads them, and then cleans it up afterwards.